src/pages/ForgotPassword/Index.tsx

```diff
@@ -33,9 +33,17 @@ export default function ForgotPassword() {
     });
   };
 
+  const error = () => {
+    Modal.error({
+      title: t("ForgotPassword.emailNotFound"),
+      centered: true,
+    });
+  };
+
   async function onSubmit({ email }: ForgotPasswordForm) {
-    await forgotPassword({ email });
-    success();
+    const { data } = (await forgotPassword({ email })) as { data: boolean };
```

Comment:
why we need 'as'?

Reply:
![image](https://user-images.githubusercontent.com/76705901/186332828-b155c51c-7871-4aa3-9010-78485ae19da8.png)

Because without type assertion it throw error. I didn't find out another way to fix it

Reply:
const { data }: {data: boolean} = await forgotPassword({ email })
no?

Reply:
![image](https://user-images.githubusercontent.com/76705901/186752240-38c1c3df-136a-41fc-845d-f17a76fac0f6.png)

Reply:
point me to forgotPassword function. maybe types inside described bad

Reply:
![image](https://user-images.githubusercontent.com/76705901/186825989-571bd0a2-7d59-4533-a122-2a422e88d9e3.png)